service/src/main/frontend/src/pages/DataDictionary.vue

```diff
@@ -0,0 +1,295 @@
+                @click="queryTable"
+              />
+        </template>
+
+        <template v-slot:header="props">
+          <q-tr :props="props">
+            <q-th />
+            <q-th style="font-size: 13.7px;" v-for="col in props.cols" :key="col.name" :props="props">
+              {{ col.label }}
+            </q-th>
+          </q-tr>
+        </template>
+
+        <template v-slot:body="props">
+          <q-tr
+            :props="props"
+            v-if="Formatters.isVisible(props.row)"
+          >
+            <q-td style="width: 60px; min-width: 60px">
+              <q-btn
+                size="9px"
+                color="cyan-8"
+                round
+                dense
+                @click="
+                  {
+                    props.expand = !props.expand;
+                    Formatters.toggleVisibility(props.row);
+                  }
+                "
+                :icon="props.row.isVisible.value ? 'remove' : 'add'"
+                v-if="Formatters.buttonParse(props.cols, props.row)"
+              />
+              <q-icon
+                  style="margin-left: 4px;"
+                  size="1rem"
+                  :name="'bi-arrow-right'"
+                  color="cyan-8"
+                  v-if="props.row.duplicate == 1"
+              />
+            </q-td>
+            <q-td
+              v-for="col in props.cols"
+              :key="col.name"
+              :props="props"
+              style="font-size: 13px;"
+              :title="Formatters.parseVal(col.name, col.value)"
+              @click="Feature.copyLabel(col.value)"
+            >
+              <label style="cursor: pointer;">
+                {{
+                Formatters.maxSubstring(
+                  Formatters.parseVal(col.name, col.value), col.name
+                )
+              }}
+              </label>
+            </q-td>
+          </q-tr>
+        </template>
+      </q-table>
+    </div>
+  </main>
+  <div v-if="banner?.enabled" :style="banner?.styleTop" style="margin-top: 0.50vh;">
+      {{ banner?.messageTop }}
+  </div>
+  <div v-if="banner?.enabled" :style="banner?.styleBottom">
+      {{ banner?.messageBottom }}
+  </div>
+</template>
+
+<script setup lang="ts">
+import { onMounted, ref } from 'vue';
+import { QTable, QTableProps, exportFile, useQuasar, Notify } from 'quasar';
+import { useToggle, useDark } from '@vueuse/core';
+import { api } from '../boot/axios';
+import { Banner, columns } from '../functions/components';
+import * as Formatters from '../functions/formatters';
+import * as Wrapper from '../functions/csvWrapper';
+import * as Feature from '../functions/features';
+
+// Defines the Table References, loading for axios, search filter, and pagination to sort.
+const $q = useQuasar();
+const table = ref();
+const loading = ref(true);
+const filter = ref('');
+const changeFilter = ref('');
+const banner = ref<Banner>();
+let rows: QTableProps['rows'] = [];
+let paginationFront = ref({
```

Comment:
In general I'd default to `const` over `let`. This variable doesn't seem to be reassigned, nor should it. Let the compiler help us out here in the future and make it a constant.
```suggestion
const paginationFront = ref({
```